services/src/main/java/org/exoplatform/news/service/NewsService.java

```diff
@@ -75,9 +75,10 @@ public interface NewsService {
    *
    * @param news to be published
    * @param publisher of the News
+   * @param broadcast if to broadcast listener
    * @throws Exception when error
    */
-  void publishNews(News news, String publisher) throws Exception;
+  void publishNews(News news, String publisher, boolean broadcast) throws Exception;
```

Comment:
We should not change the API in a maintenance task, could you add another function with the new parameter ?

Reply:
done!